CMSgov/design-system
# [WNMGDS-1687] NewRelic monitoring for doc site

## Summary

https://jira.cms.gov/browse/WNMGDS-1687

### Added

- Added NewRelic monitoring agent in Gatsby site

### Changed

- Changed node version from `14.17` to `16.15.1`, which `gatsby-plugin-newrelic` wanted
  - Which also meant we needed to upgrade `gulp-sass` because the version of `node-sass` that `gulp-sass` required was outdated and didn't support node 16. Even though our scripts package was installing the newer version of `node-sass` (v7), that gulp plugin wanted an older one.

- [x] Need to update our Jenkins scripts

## How to test

1. `yarn build-storybook:gatsby && yarn build:gatsby && yarn serve:gatsby`
2. Log into NewRelic and view our browser application. You should see your traffic being reported

<img width="907" alt="Screen Shot 2022-06-21 at 7 50 06 PM" src="https://user-images.githubusercontent.com/7595652/174934440-58ae7157-1839-4ae9-b8bb-d536d99abd1c.png">
